tracepointdebug/probe/log_point_manager.py

```diff
@@ -92,6 +95,21 @@ def expire_log_point(self, log_point):
                 self._log_points.pop(log_point_id).remove_log_point()
 
     def publish_event(self, event):
+        try:
+            if self.broker_manager._log_data_redaction_callback and isinstance(event, LogPointEvent):
+                log_redaction = {
+                    "file_name": event.file,
+                    "line_no": event.line_no,
+                    "method_name": event.method_name,
+                    "log_message": event.log_message,
+                }
+                self.broker_manager._log_data_redaction_callback(log_redaction)
+                event.file = log_redaction["file_name"]
+                event.line_no = log_redaction["line_no"]
+                event.method_name = log_redaction["method_name"]
```

Comment:
Do we expect masking method name so we are getting it again from redacted trace?